lib/convert.ts

```diff
@@ -36,20 +31,20 @@ export default class Convert {
    *   to deal with http/https sources in the future.
    */
   public static uriToPath(uri: string): string {
-    const url = URL.parse(uri);
```

Comment:
URL.parse is deprecated

Reply:
@UziTech If we fix this, we can enable the strict variant of the config

Reply:
Can't we just change it to `new URL(uri)`?

Reply:
`url.path` does not exist in the new API. There is `pathname`, but I am not sure if it is the same.

Reply:
according to the [docs](https://nodejs.org/api/url.html#url_urlobject_path): `The path property is a concatenation of the pathname and search components.`

Reply:
So, I think it is fixed in 
https://github.com/atom-community/atom-languageclient/pull/125/commits/358d254391847077309e9e1170156a94e8d4f27a

Reply:
Strangely, the tests only pass on Windows, and they fail on the Unix systems

Reply:
I say we just add `/* eslint-disable node/no-deprecated-api */` at the top of the file for now and fix it in another PR.

Reply:
It seems like it is failing because of [this](https://stackoverflow.com/questions/59375013/node-legacy-url-parse-deprecated-what-to-use-instead) so we will have to figure out some other fix than just using the new URL api.

Reply:
I made the error a warning for now, so we can merge this PR.